_i18n/de/resources/user-guides/monero-wallet-cli.md

```diff
@@ -1,147 +1,113 @@
 
-### Sending to a standard address:
+### An eine Standardadresse senden
 
     transfer ADDRESS AMOUNT PAYMENTID
 
-Replace `ADDRESS` with the address you want to send to, `AMOUNT` with how many monero you want to send,
-and `PAYMENTID` with the payment ID you were given. Payment ID's are optional. If the receiving party doesn't need one, just
-omit it.
+Ersetze `ADDRESS` mit der Adresse, an die du senden möchtest, `AMOUNT` mit dem Betrag der Monero, die du versenden wirst und `PAYMENTID` mit der dir bereitgestellten Zahlungs-ID. Zahlungs-IDs sind optional: Wenn der Empfänger keine benötigt, kannst du sie einfach weglassen.
 
-### Sending to an integrated address:
+### An eine integrierte Adresse senden
 
     transfer ADDRESS AMOUNT
 
-The payment ID is implicit in the integrated address in that case.
+In diesem Fall ist die Zahlungs-ID in die integrierte Adresse inkludiert.
 
-### Specify the number of outputs for a transaction:
+### Die Anzahl an Outputs für eine Transaktion festlegen
 
     transfer RINGSIZE ADDRESS AMOUNT
 
-Replace `RINGSIZE` with the number of outputs you wish to use. **If not specified, the default is 7.** It's a good idea to use the default, but you can increase the number if you want to include more outputs. The higher the number, the larger the transaction, and higher fees are needed.
-
+Ersetze `RINGSIZE` mit der Anzahl an Outputs, die du verwenden möchtest. **Die Standardeinstellung ist 11, wenn nichts anderes festgelegt wird.** Es ist ratsam, die voreingestellte Ringgröße zu verwenden; du kannst die Zahl aber auch erhöhen, wenn du mehr Outputs beifügen möchtest. Je größer die Zahl ist, desto größer ist auch die Transaktion und die mit ihr einhergehenden Gebühren.
 
-## Receiving monero
+## Monero empfangen
 
-If you have your own Monero address, you just need to give your standard address to someone.
+Wenn du eine eigene Monero-Adresse hast, kannst du dem an dich Sendenden ganz einfach deine Standardadresse geben.
 
-You can find out your address with:
+Diese findest du heraus mit:
 
     address
 
-Since Monero is anonymous, you won't see the origin address the funds you receive came from. If you
-want to know, for instance to credit a particular customer, you'll have to tell the sender to use
-a payment ID, which is an arbitrary optional tag which gets attached to a transaction. To make life
-easier, you can generate an address that already includes a random payment ID:
+Da Monero anonym ist, wirst du die Ursprungsadresse deiner empfangenen Gelder nicht sehen können. Wenn du sie aber doch aus irgendeinem Grund (beispielsweise, um einen besonderen Kunden zu honorieren) wissen möchtest, musst du dem Sender mitteilen, dass er eine Zahlungs-ID verwenden soll. Diese ist ein willkürlicher, optionaler Tag, mit welchem die Transaktion versehen wird. Um es dir einfach zu machen, kannst du eine Adresse erstellen, die bereits eine zufällige Zahlungs-ID enthält:
```

Comment:
Not sure what "to credit a particular customer" means and if "einen besonderen Kunden honorieren" is the correct translation?

Reply:
I had my troubles with translating this very part. I couldn'd find a better German word than "honorieren" which could be used in this context (in my opinion, as I understood that you want to give back something to a good and well-paying customer, but I may be wrong of course). It may sound a bit stilted, so I'm up to ideas! :)

Reply:
@erciccione

Reply:
As I understood it it is not about some "special, good and well-paying customer" but about all customers. Every customer's order gets its own unique payment id, otherwise the seller cannot relate payments to orders.